scripts/setup-macos.sh

```diff
@@ -169,4 +174,4 @@ function install_velox_deps {
 )
 
 echo 'To add cmake-format bin to your $PATH, consider adding this to your ~/.profile:'
-echo 'export PATH=$HOME/bin:$HOME/Library/Python/3.7/bin:$PATH'
+echo 'Please add $INSTALL_PREFIX to your ~/.zshrc'
```

Comment:
Does this gel with the preceding line?  Looks like it is a bit disjointed.
Should maybe be
```
export INSTALL_PREFIX=<path/to/installed/dependencies>
```
As used when the setup scripts are used?

Is this expanded in the output?

We probably should state
```
export INSTALL_PREFIX=$INSTALL_PREFIX
```
to your .zshrc?

Reply:
I removed the cmake-format echo message because the Python version is not 3.7 anymore. We use pip now and it should install to a standard path.

I fixed the message for INSTALL_PREFIX. We need to use double quotes to expand the variable. Single quotes will be treated as text.